integration-tests/test-cases/scenario-tests/src/test/java/io/cellery/integration/scenario/tests/ObservabilityDashboard.java

```diff
@@ -0,0 +1,365 @@
+                "/div/table/tbody/tr[1]/td/div";
+        WebElement component = refreshedPage.findElement(By.xpath(componentFieldXPath));
+        Assert.assertEquals(component.getText(), "proxy");
+
+        String sequenceDiagramButtonXPath = "//*[@id=\"root\"]/div/main/div[3]/div[1]/div/div/div/button[2]";
+        clickOnObservabilityButton(sequenceDiagramButtonXPath);
+        checkSeqDiagram();
+
+        String dependancyDiagramButton = "//*[@id=\"root\"]/div/main/div[3]/div[1]/div/div/div/button[3]";
+        clickOnObservabilityButton(dependancyDiagramButton);
+
+        String dependancyDiagramXPath = "//*[@id=\"root\"]/div/main/div[3]/div[2]/div/div/div/canvas";
+        webDriver.findElement(By.xpath(dependancyDiagramXPath));
+
+    }
+
+    private void checkSeqDiagram() {
+        for (Map.Entry<String, Integer> entry : diagram.getSequenceDiagramCalls().entrySet()) {
+            checkSeqDiagramCell(entry.getKey(), entry.getValue());
+        }
+        for (Map.Entry<String, Integer> entry : diagram.getSequenceDiagramCells().entrySet()) {
+            webDriver.findElement(By.xpath("//*[name()='svg']/*[name()='g'][" + entry.getValue() +
+                    "]/*[name()" +
+                    "='text']")).click();
+            String firstComp =
+                    webDriver.findElement(By.xpath("//*[name()='svg']/*[name()='g'][2]/*[name()" +
+                            "='text']/*[name()" +
+                            "='tspan']")).getText();
+            Assert.assertEquals(firstComp, entry.getKey());
+        }
+
+    }
+
+    private void checkSeqDiagramCell(String instanceName, int cellSlotNumber) {
+        String cellName =
+                webDriver.findElement(By.xpath("//*[name()='svg']/*[name()='g'][" + cellSlotNumber +
+                        "]/*[name()" +
+                        "='text']/*[name()" +
+                        "='tspan']")).getText();
+        Assert.assertEquals(cellName, instanceName.replace("-", "_"));
+    }
+
+    public void loginObservability() {
+        webDriver.get(Constants.DEFAULT_OBSERVABILITY_URL);
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("/html/body/header/div/div/a/img")));
+        WebElement username = webDriver.findElement(By.id("username"));
+        WebElement password = webDriver.findElement(By.id("password"));
+        username.sendKeys("admin");
+        password.sendKeys("admin");
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@id=\"loginForm\"]/div[6" +
+                "]/div/button"))).click();
+        try {
+            wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("approveCb"))).click();
+            wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("consent_select_all"))).click();
+            wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("approve"))).click();
+        } catch (Exception ignored) {
```

Comment:
Why do we ignore the exception here?

Reply:
If the user has already logged in to observability portal before, this screen won't appear. Since a new cluster is  made whenever this test is ran on the server its ok to get rid of this try catch, however, it will be hard for testing in our local environments.  What do you think should we get rid of this try catch?